src/AccessibilityInsights.Extensions.AzureDevOps/Extensions.AzureDevOps.csproj

```diff
@@ -131,6 +131,10 @@
     <Reference Include="System.Net.Http.Formatting, Version=5.2.6.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.AspNet.WebApi.Client.5.2.6\lib\net45\System.Net.Http.Formatting.dll</HintPath>
     </Reference>
+    <Reference Include="System.Runtime, Version=4.1.1.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Is there a reason this entry did not already exist in the .csproj?

Reply:
In the past, I think I manually trimmed unused references. The VS tooling automatically adds all dependencies from the package without determining that any code actually references the assembly being added